package-res/ccc/core/base/chart/chart.axes.js

```diff
@@ -534,8 +628,14 @@ pvc.BaseChart
 
     _onColorAxisScaleSet: function(axis) {
         switch(axis.index) {
-            case 0: this.colors = axis.scheme(); break;
-            case 1: if(this._allowV1SecondAxis) this.secondAxisColor = axis.scheme(); break;
+            case 0: this.colors = axis.scheme(); 
```

Comment:
These changes here shouldn't be needed, if state is built upon request, by calls to `axis.getState`.